Feat: Use skinTone in emoji shortcodes

emschorsch opened the pull request:
To enable this, brought in a package that supports checking if a given emoji supports a shortcode or not. Follow-up now that we're [persisting](https://github.com/TryQuiet/quiet/pull/2938) the skinTone.


### Pull Request Checklist

- [x] I have linked this PR to a related [GitHub issue](https://github.com/TryQuiet/quiet/issues/2794).
- [x] I have added a description of the change (and Github issue number, if any) to the root [CHANGELOG.md](https://github.com/TryQuiet/quiet/blob/develop/CHANGELOG.md).

### (Optional) Mobile checklist

Please ensure you completed the following checks if you did any changes to the mobile package:

- [ ] I have run e2e tests for mobile
- [ ] I have updated base screenshots for visual regression tests

Commit:
Feat: Use skinTone in emoji shortcodes

Brought in a package that supports checking if a given emoji supports a shortcode or not.

Commit:
Feat: Use skinTone in emoji shortcodes

Brought in a package that supports checking if a given emoji supports a shortcode or not.

holmesworcester reviewed (approved):
This looks great! Thank you!

I just approved the test run so assuming all tests pass this is good to merge.

I do think it would be helpful to have two tests to protect against regressions from future changes.  One test could confirm skintone persists, and the other test could confirm that it applies to emojicodes.

This would be a good way of getting to know our testing frameworks as well.

Commit:
Feat: Use skinTone in emoji shortcodes

Brought in a package that supports checking if a given emoji supports a shortcode or not.

holmesworcester commented:
These test failures look like they're related to a recent problem we've been having with tests and not related to the PR.

Thank you for your work on this @emschorsch! Merging!
